buster/retriever/sqlite.py

```diff
@@ -44,10 +44,10 @@ def __del__(self):
         if self.db_path is not None:
             self.conn.close()
 
-    def get_documents(self, source: str) -> pd.DataFrame:
+    def get_documents(self, source: str = None) -> pd.DataFrame:
         """Get all current documents from a given source."""
         # Execute the SQL statement and fetch the results.
-        if source == "":
+        if source is None:
```

Comment:
You want to keep the `source == ""` here too.

Reply:
why ?

Reply:
what happens if the source doesnt exist? does it also crash?

Reply:
nope it just returns nothing